src/client/components/line-chart-axis/line-chart-axis.tsx

```diff
@@ -31,6 +31,13 @@ export class TimeAxis extends React.Component<TimeAxisProps, TimeAxisState> {
     var format = scale.tickFormat();
 
     var timezoneString = timezone.toString();
+
+    function formatLabel(v: Date | number) {
+      if (v instanceof Date) { return formatWithTimezone(v); }
+      var f = d3.format(".1f");
```

Comment:
I am pretty sure you could extract this into a constant (not call it on every function call)